demo/web/src/main/notebooks/C. Python/C2 Formulas in Python.md

```diff
@@ -0,0 +1,68 @@
+# **Python functions in Deephaven**
+
+This notebook will demonstrate how to write a Python function that can be used in the Deephaven Query Language.
+
+In this example, a custom, user-defined function is used inside a query string to compute new column values using [`update`](https://deephaven.io/core/docs/reference/table-operations/select/update/).
```

Comment:
because this is a demo notebook, and not strictly a how-to, can we give generic advice on why functions are useful?